css/main.css

```diff
@@ -480,20 +480,20 @@ body #wrapper {
       body #wrapper #second-section #background-girl-image-3 .third-article h3 {
         font-size: 26px;
         line-height: 28px; }
-  body #wrapper #second-section .container .row .col-md-offset-6 h2 {
+  body #wrapper #second-section .container .row .column-1 h2 {
```

Comment:
use this form of css blocks:

```
body #wrapper #second-section .container .row .column-1 h2 {
...
}
```

instead of:

```
body #wrapper #second-section .container .row .column-1 h2 {
... }
```

because it is easier to read.